build(docs-infra): update Angular CLI/framework/Material to v14.0.0-rc.0 in AIO and docs examples

gkalpak opened the pull request:
Update Angular CLI/framework/Material to v14.0.0-rc.0 in AIO and docs examples.
See individual commits for details.

mary-poppins commented:
You can preview 1111075 at https://pr45997-1111075.ngbuilds.io/.

mary-poppins commented:
You can preview 5bec932 at https://pr45997-5bec932.ngbuilds.io/.

mary-poppins commented:
You can preview e1f1135 at https://pr45997-e1f1135.ngbuilds.io/.

mary-poppins commented:
You can preview 79b63e9 at https://pr45997-79b63e9.ngbuilds.io/.

mary-poppins commented:
You can preview 8d3d201 at https://pr45997-8d3d201.ngbuilds.io/.

mary-poppins commented:
You can preview e2ae5c0 at https://pr45997-e2ae5c0.ngbuilds.io/.

mary-poppins commented:
You can preview 8948a7e at https://pr45997-8948a7e.ngbuilds.io/.

mary-poppins commented:
You can preview 140d855 at https://pr45997-140d855.ngbuilds.io/.

atscott reviewed (approved):
reviewed-for: size-tracking

jessicajaniuk reviewed (approved):
reviewed-for: size-tracking

gkalpak committed:
build(docs-infra): update AIO payload sizes

Update the AIO payload sizes to better see the impact in size of changes
in subsequent commits.

gkalpak committed:
build(docs-infra): update Angular CLI/framework to v14.0.0-rc.0 in docs examples

Update the docs examples to the latest prerelease versions of Angular
CLI and framework (v14.0.0-rc.0).

Also update the example apps to more closely align with new CLI apps.
See also the [diff][1] between a basic v13.0.0-rc.3 CLI app and a
v14.0.0-rc.0 one.

[1]: https://github.com/cexbrayat/angular-cli-diff/compare/13.0.0-rc.3...14.0.0-rc.0

gkalpak committed:
build(docs-infra): update Angular CLI/framework to v14.0.0-rc.0 in AIO

Update angular.io to the latest prerelease versions of Angular CLI and
framework (v14.0.0-rc.0).

Also update angular.io to more closely align with new CLI apps. See also
the [diff][1] between a basic v13.0.0-next.9 CLI app and a v14.0.0-rc.0
one.

[1]: https://github.com/cexbrayat/angular-cli-diff/compare/13.0.0-next.9...14.0.0-rc.0

gkalpak committed:
build(docs-infra): update Angular Material to v14.0.0-rc.0 in AIO

Update angular.io to the latest prerelease version of Angular Material
(v14.0.0-rc.0).

mary-poppins commented:
You can preview d09b90a at https://pr45997-d09b90a.ngbuilds.io/.

alxhub commented:
This PR was merged into the repository by commit f83ee69e17544bd69de0e9e6903c9a790c6f807d.
